dxchange/reader.py

```diff
@@ -651,40 +654,71 @@ def read_dx_meta(file_name, label1='/measurement/', label2='/process/') :
     """
     meta = {}
 
-    fp = h5py.File(file_name, 'r') 
+    fp = h5py.File(file_name, 'r')
     read_hdf5_item_structure(meta, fp, file_name, label1, label2)
     fp.close()
 
     return meta
 
 
+def create_standard_dx_meta(file_name):
```

Comment:
At the call today, @michael-sutherland mentioned pydantic. If I caught his comment correctly, then we could use it to better define the data model of the return. In this scenario, the signature might use the type hint:

``` python
def create_standard_dx_meta(file_name) -> DXMeta:
```

Where DXMeta could be a pytdantic class:

``` python
class DXMeta(pydantic.BaseModel):
  pixel_size: int,
  pixel_size_units: int,
  binning_x: int,
  energy: float,
  energy_units: int,
  exposure_time: float,
  time_stamp: datetime,
  uuid: str
```

This generally adds complexity (and a new dependency on the project) but is very readable and explicit. It also excels for cases where an API is taking data as input because it provides built in validation, but this is the return value. So, there's a tradeoff that should be considered.